contracts/modules/Checkpoint/Automation/ScheduleCheckpointStorage.sol

```diff
@@ -0,0 +1,27 @@
+pragma solidity 0.5.8;
+
+contract ScheduleCheckpointStorage {
+
+    enum FrequencyUnit { SECONDS, DAYS, WEEKS, MONTHS, QUATER, YEARS }
+    
+    bytes32 constant OPERATOR = "OPERATOR";
+
+    struct Schedule {
+        uint256 index;
+        bytes32 name;
+        uint256 startTime;
+        uint256 endTime;
+        uint256 nextCheckPointCreatedAt;
```

Comment:
`nextCheckPointCreatedAt` is confuing. Rename to `createNextCheckpoint` maybe?
Then add a code comment in the update function that since checkpoint has been created, this is now being treated as `lastCheckpointCreatedAt`.